LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
Why did you remove LICENSE?

Reply:
For lack of attention I ended up deleting the license file